src/main/scala/com/typesafe/sbt/packager/archetypes/JavaAppBatScript.scala

```diff
@@ -11,9 +11,10 @@ object JavaAppBatScript {
 
   def makeWindowsRelativeClasspathDefine(cp: Seq[String]): String = {
     def cleanPath(path: String): String = path.replaceAll("/", "\\\\")
+    def isAbsolute(path: String): Boolean = path.length > "c:\\".length && Character.isLetter(path(0)) && path(1) == ':'
```

Comment:
I'm a bit confused about the `"c:\\".length`definition. It looks like putting documentation into the source code.
If you have to check for a path longer than 4 characters do so explicitly and add a small comment for what reason. At first I thought you were checking if the path starts with `c:\\`` ;)